### PR TITLE
Reorganize addons in GSRI Gear

### PR DESCRIPTION
**Pull request by ImplicitDeny**
Merging all small addons into 3 consistent ones (including a temporary one).

Also perform a bazilion small tweaks, fixes, enhancements etc.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -27,69 +27,29 @@ jobs:
     - name: Checkout files
       uses: actions/checkout@v2
 
-    - name: Build GSRI Backpacks
+    - name: Build GSRI Equipment
```
Comment (ArwynFr) [inline .github/workflows/release.yml:30]: What dependencies are we breaking by changing the pboprefix of addons ?

File: addons/gsri_equipment/weapons_compat.hpp [hunk 2]
```diff
@@ -0,0 +1,17 @@
+class Rifle;
+class Rifle_Base_F : Rifle {
+	class WeaponSlotsInfo;
+};
+class arifle_MSBS65_base_F : Rifle_Base_F {
+	magazineWell[] += {"CBA_556x45_STANAG"};
```
Comment (ArwynFr) [inline addons/gsri_equipment/weapons_compat.hpp:6]: Is your new MSBS compatible with both 6.5mm and 5.56mm ?

File: addons/gsri_props/config.cpp [hunk 3]
```diff
@@ -0,0 +1,97 @@
+class CfgPatches {
+  class gsri_lockers {
```
Comment (ArwynFr) [inline addons/gsri_props/config.cpp:2]:
```suggestion
  class gsri_props {
```

**Review by PhileasLX — DISMISSED** [review]
Good for me

**Review by ArwynFr — CHANGES_REQUESTED** [review]
I would prefer content paths to match `./addons/gsri_equipment/backpacks/Data` rather than `./addons/gsri_equipment/Data/backpacks`. Also `backpacks.hpp` could move to `backpacks/config.cpp`. This makes the addon more modular IMHO.

File: addons/gsri_equipment/weapons_compat.hpp (hunk 2 above)
Comment (ImplicitDeny) [reply addons/gsri_equipment/weapons_compat.hpp:6]: Yes, I only add the 5.56 mag well to the weapon, not taking existing ones out.

File: .github/workflows/release.yml (hunk 1 above)
Comment (ArwynFr) [reply .github/workflows/release.yml:30]: Merging this PR requires a breaking change convention

**Review by ArwynFr — APPROVED** [review]
LGTM but beware the breaking change !

**Pull request merged**